Week1/KOUOKAM DIDERIQUE/stairecaseFunction.js

```diff
@@ -0,0 +1,13 @@
+/* This function prints the staircase of size n.*/
+function staircase(n) {
+    let x = new Array(n).fill(' ');
+    
+    while (x.indexOf(' ') !==-1) {
+        x.shift();
+        x.push('@');
```

Comment:
You used `@` symbol to print your staircase. The question clearly states using `#`